frontend/portals/ProductVideo/component.jsx

```diff
@@ -1,10 +1,12 @@
 import React, { Component } from 'react';
```

Comment:
React reports and error when selecting a different product variant. It does not have to be a product with a video. This happens on both the GMD and ios11 themes.

![image](https://user-images.githubusercontent.com/11764719/52582787-a3356880-2df3-11e9-9c88-64aded1dfd89.png)

Reply:
Good catch. Fixed it.